src/test/suite/views/webview-app/components/form/radio-box-group/radio-box-group.test.tsx

```diff
@@ -0,0 +1,45 @@
+import assert from 'assert';
+import * as React from 'react';
+import { mount } from 'enzyme';
+
+import RadioBoxGroup from '../../../../../../../views/webview-app/components/form/radio-box-group/radio-box-group';
+
+describe('Connect Form Modal Component Test Suite', () => {
```

Comment:
This is probably a copy-and-paste leftover - should be `Radio Box Group Component Test Suite`? 🤔